asv_bench/benchmarks/rolling.py

```diff
@@ -252,4 +252,21 @@ def time_groupby_mean(self, engine):
         self.gb_ewm.mean(engine=engine)
 
 
+def table_method_func(x):
+    return np.sum(x, axis=0) + 1
+
+
+class TableMethod:
+
+    params = ["single", "table"]
+    param_names = ["method"]
+
+    def setup(self, method):
+        df = pd.DataFrame(np.random.randn(10, 1000))
+        self.roll = df.rolling(2, method=method)
+
+    def time_apply(self, method):
```

Comment:
is method not used in this one?  so we're re-doing the same benchmark twice?  (not a deal-breaker asi think this happens in a number of places)

Reply:
Good point. I think I put the parameterization in the wrong place. Thanks